aws_lambda_builders/workflows/rust_cargo/actions.py

```diff
@@ -0,0 +1,148 @@
+"""
+Rust Cargo build actions
+"""
+
+import os
+import subprocess
+import shutil
+import json
+
+from aws_lambda_builders.workflow import BuildMode
+from aws_lambda_builders.actions import ActionFailedError, BaseAction, Purpose
+
+DEFAULT_CARGO_TARGET = "x86_64-unknown-linux-gnu"
+
+
+class BuilderError(Exception):
```

Comment:
I wonder if we should name these classes with `Rust` prefix? `RustBuilderError` for instance.